floor/floor/controller/midi/functions.py

```diff
@@ -90,29 +90,31 @@ def add(cls, name, **kwargs):
                   callback=lambda controller, _: controller.playlist.go_to(16),
```

Comment:
I accepted several suggested changes from PyCharm about the formatting of this file.